app/views/comments/_comment.html.erb

```diff
@@ -3,6 +3,9 @@
     <p class="comment-author"><%= comment.user.first_name %></p>
   </div>
   <div class="comment-body clearfix">
-    <%= comment_markdown(comment.text) %>
+    <%= best_in_place [comment.card, comment], :text, as: :textarea,
+                      display_with: -> (text) { comment_markdown(text) },
+                      ok_button: t("comments.edit.save"),
```

Comment:
The problem with not having a button on best_in_place multiline inputs is basically that the UX gets confusing: what should you do to save? Exiting the area? You cannot even press Enter, as it would enter a newline.

So, to answer your question: probably yes, but I'd bet GitHub did it for the same reason :ok_hand: